utils/web3/unreal.ts

```
@@ -0,0 +1,149 @@
+
+  let txHash: Address
+  try {
+    txHash = await wallet.writeContract({
+      address: token,
+      abi: unrealTokenAbi,
+      functionName: "transfer",
+      args: [to, amount],
+      account: wallet.account,
+      chain: wallet.chain,
+    })
+  } catch (err: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)